chorus_thing/database.py

```diff
@@ -19,3 +19,20 @@ class BaseTable(AbstractConcreteBase, ChorusThing):
     id: Mapped[int] = mapped_column(primary_key=True)
     created: Mapped[datetime] = mapped_column(default=datetime.now(tz=UTC))
     modified: Mapped[datetime] = mapped_column(default=datetime.now(tz=UTC), onupdate=datetime.now(tz=UTC))
+
+
+class Event(BaseTable):
+    """Table of Choral Events."""
+
+    __tablename__ = "event"
+
+    paid_event: Mapped[bool] = mapped_column(default=False)
+    location: Mapped[str]
+    event_time: Mapped[datetime] = mapped_column(default=datetime.now(tz=UTC), onupdate=datetime.now(tz=UTC))
```

Comment:
```suggestion
    event_time: Mapped[datetime | None]
```

remove the mapped_column and make this optinal